### ISSUE TITLE
Problems with @load *.jld files 

### ISSUE THREAD

**Issue by alexlyapin** [JuliaIO/JLD.jl#103]
Got a bug during loading *.jld file. The basic code from JLD description works fine. 
Mine .jld file is compiled via OS X and I'm running my code on Win platform.
```
using JLD
data= load("d:\\Folder\\file.jld")
```
Here is bug log:

```
Please submit a bug report with steps to reproduce this fault, and any error messages that follow (in their entirety). Thanks.
Exception: EXCEPTION_ACCESS_VIOLATION at 0x7ffd05380b7d --

RtlFreeHeap at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
while loading console, in expression starting on line 3
RtlFreeHeap at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
RtlFreeHeap at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
free at C:\WINDOWS\system32\msvcrt.dll (unknown line)
h5t_get_member_name at C:\Users\AlexLPN\.julia\v0.5\HDF5\src\plain.jl:2188
jldatatype at C:\Users\AlexLPN\.julia\v0.5\JLD\src\jld_types.jl:735
read at C:\Users\AlexLPN\.julia\v0.5\JLD\src\JLD.jl:390
unknown function (ip: 000000002110A276)
read at C:\Users\AlexLPN\.julia\v0.5\JLD\src\JLD.jl:366
collect_to! at .\array.jl:340
collect_to_with_first! at .\array.jl:327
unknown function (ip: 0000000021113F79)
jl_call_method_internal at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\julia_internal.h:189 [inlined]
jl_apply_generic at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\gf.c:1942
collect at .\array.jl:308
#37 at C:\Users\AlexLPN\.julia\v0.5\JLD\src\JLD.jl:1224
unknown function (ip: 0000000021108966)
jl_call_method_internal at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\julia_internal.h:189 [inlined]
jl_apply_generic at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\gf.c:1942
#jldopen#11 at C:\Users\AlexLPN\.julia\v0.5\JLD\src\JLD.jl:265
load at C:\Users\AlexLPN\.julia\v0.5\JLD\src\JLD.jl:1223
unknown function (ip: 00000000210FF8C6)
jl_call_method_internal at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\julia_internal.h:189 [inlined]
jl_apply_generic at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\gf.c:1942
#load#13 at C:\Users\AlexLPN\.julia\v0.5\FileIO\src\loadsave.jl:45
load at C:\Users\AlexLPN\.julia\v0.5\FileIO\src\loadsave.jl:45
jl_call_method_internal at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\julia_internal.h:189 [inlined]
jl_apply_generic at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\gf.c:1942
do_call at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\interpreter.c:66
eval at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\interpreter.c:190
eval_body at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\interpreter.c:469
jl_interpret_call at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\interpreter.c:573
jl_toplevel_eval_flex at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\toplevel.c:572
jl_parse_eval_all at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\ast.c:717
include_string at .\loading.jl:441
unknown function (ip: 0000000061E2DEB5)
jl_call_method_internal at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\julia_internal.h:189 [inlined]
jl_apply_generic at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\gf.c:1942
do_call at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\interpreter.c:66
eval at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\interpreter.c:190
eval_body at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\interpreter.c:469
jl_interpret_call at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\interpreter.c:573
jl_toplevel_eval_flex at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\toplevel.c:572 [inlined]
jl_toplevel_eval at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\toplevel.c:580
jl_toplevel_eval_in_warn at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\builtins.c:590
eval at .\boot.jl:234
unknown function (ip: 0000000061C4D605)
jl_call_method_internal at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\julia_internal.h:189 [inlined]
jl_apply_generic at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\gf.c:1942
#60 at C:\Users\AlexLPN\.julia\v0.5\Atom\src\eval.jl:39
withpath at C:\Users\AlexLPN\.julia\v0.5\CodeTools\src\utils.jl:30
unknown function (ip: 00000000210C4E26)
jl_call_method_internal at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\julia_internal.h:189 [inlined]
jl_apply_generic at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\gf.c:1942
withpath at C:\Users\AlexLPN\.julia\v0.5\Atom\src\eval.jl:45
unknown function (ip: 00000000210C4A86)
macro expansion at C:\Users\AlexLPN\.julia\v0.5\Atom\src\eval.jl:107 [inlined]
#59 at .\task.jl:60
jl_call_method_internal at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\julia_internal.h:189 [inlined]
jl_apply_generic at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\gf.c:1942
jl_apply at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\julia.h:1392 [inlined]
start_task at /home/Administrator/buildbot/slave/package_win6_2-x64/build/src/home/Administrator/buildbot/slave/package_win6_2-x64/build/src\task.c:253
Allocations: 16183035 (Pool: 16178729; Big: 4306); GC: 33
```
`

**Comment by YongHee-Kim** [comment]
Same error occurs for test script on windows7 & Julia 5.0. 66a6ce5b6c34492ecbb986dff8d318cc02b182d9

**Comment by simonster** [comment]
Probably the same as https://github.com/JuliaIO/HDF5.jl/issues/323

**Comment by simonster** [comment]
This will probably be fixed by `Pkg.update()` once https://github.com/JuliaLang/METADATA.jl/pull/6977 is merged

**Comment by simonster** [comment]
Update: now waiting for https://github.com/JuliaLang/METADATA.jl/pull/6982

**Comment by simonster** [comment]
Should be fixed now if you update to HDF5 v0.7.0. If not please comment.
